hbase-client/src/main/java/org/apache/omid/transaction/TTable.java

```diff
@@ -228,21 +261,12 @@ public void delete(Transaction tx, Delete delete) throws IOException {
                 }
             }
         }
-        if (issueGet) {
-            // It's better to perform a transactional get to avoid deleting more
-            // than necessary
-            Result result = this.get(transaction, deleteG);
-            if (!result.isEmpty()) {
-                for (Entry<byte[], NavigableMap<byte[], NavigableMap<Long, byte[]>>> entryF : result.getMap()
-                    .entrySet()) {
-                    byte[] family = entryF.getKey();
-                    for (Entry<byte[], NavigableMap<Long, byte[]>> entryQ : entryF.getValue().entrySet()) {
-                        byte[] qualifier = entryQ.getKey();
-                        deleteP.add(family, qualifier, CellUtils.DELETE_TOMBSTONE);
-                        transaction.addWriteSetElement(new HBaseCellId(table, delete.getRow(), family, qualifier,
-                                                                       transaction.getStartTimestamp()));
-                    }
-                }
+
+        if (deleteFamily) {
```

Comment:
1. This is a deletion path without read requirement. It is doable only in row level conflict detection mode where we only need to mark the row as a player for conflict analysis. 
2. TTable works only for HBase. You can see that each one of the method starts with a check that the transaction is hbase transaction. IMHO TTable should inherit from HTableInterface and override its functions, but this is for  a different patch :).